opencog/moses/moses/local_moses.cc

```diff
@@ -107,7 +108,8 @@ static bool expand_deme(metapopulation& mp,
 void local_moses(metapopulation& mp,
                  deme_expander& dex,
                  const moses_parameters& pa,
-                 moses_statistics& stats)
+                 moses_statistics& stats,
+                 AtomSpace *_as)
```

Comment:
@behailu04, I think that by adding a atomspace field in `deme_expander` you can also avoid passing the atomspace as argument of `local_moses` and a few others.

Reply:
I thought I removed those things.

Reply:
@ngeiswei I didn't quite get why it is failing though.